build/linux/installer/conf/windows_rs_containerinventory.conf

```diff
@@ -22,4 +22,11 @@
     flush_thread_count 5
     </buffer>
     keepalive true

```

Comment:
am I correct to assume that this file is inserted into rs fluentd config if AZMON_WINDOWS_FLUENT_BIT_DISABLED is disabled?

Reply:
yes